### PR TITLE
Resample bug, see #137

### PR DESCRIPTION
**Pull request by jonasvdd**
See #137 137

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -28,8 +28,9 @@ trace-updater = ">=0.0.8"
 numpy = ">=1.14"
 Flask-Cors = "^3.0.10"
 orjson = "^3.8.0"  # Faster json serialization
-Werkzeug = "2.1.2"  # Fixating werkzeug version because of:
+# TODO -> check other werkzeug versions: 2.1.2 and 2.1.1 seem to work.
```
Comment (jvdd) [inline pyproject.toml:31]: Should this be included in this PR? For me it's fine to move this to another PR

**Review by jvdd — APPROVED** [review]
LGTM! Just check my comment wrt werkzeug versioning

File: pyproject.toml (hunk 1 above)
Comment (jonasvdd) [reply pyproject.toml:31]: I think it can be included!

**Review by jonasvdd — COMMENTED** [review]
LGTM

**Comment by jvdd** [comment]
Guess the only thing blocking merging this PR is checking other werkzeug versions!

File: pyproject.toml (hunk 1 above)
Comment (jvdd) [reply pyproject.toml:31]: Will merge this PR and leave this for what it is
